src/components/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="box">
+        <div className="display">
+          <Display />
+        </div>
+        <div className="button">
```

Comment:
- [ ] [OPTIONAL] Give the class name 'button' a more descriptive name like button-container or even just buttons.